aries-test-harness/features/0011-0183-revocation.feature

```diff
@@ -99,7 +118,7 @@ Feature: Aries agent credential revocation and revocation notification RFC 0011
       Examples:
          | issuer | credential_data   | timeframe       | request_for_proof              | presentation                  |
          | Acme   | Data_DL_MinValues | -86400:+86400   | proof_request_DL_revoc_address | presentation_DL_revoc_address |
-         | Acme   | Data_DL_MinValues | -604800:now       | proof_request_DL_revoc_address | presentation_DL_revoc_address |
+         | Acme   | Data_DL_MinValues | -604800:now     | proof_request_DL_revoc_address | presentation_DL_revoc_address |
```

Comment:
Per the "best practices guidance" that we have defined, proof requests should NEVER use a range of values. They should only use a single point in time -- either "now()" or some time in the past.  So I think you should eliminate the example that has a range.

Reply:
Are you sure we should remove those original timeframes? These work, and someone already may be using the intervals like this. Keeping them there may at least capture any desired or undesired change to revocation. I understand that if everyone follows the best practices in 441 that these will not be needed, but maybe we should wait until 441 is out there?

Reply:
I think a better action would be to ADD cases only using one time boundary: the new code in aca-py (that will be merged) should handle things no matter what, and adjust the `to` `from` values to what is allowed/desired.

Reply:
@swcurran  I'm resolving this conversation since changing the ranges is based on 441, which is not submitted yet. The state of the tests against such changes is in flux, so I don't know what to effectively change them to. My advice is to submit this and worry about the interval moving to an instant when 441 is done. Thoughts?